twake/frontend/src/app/scenes/Apps/Messages/Input/Input.tsx

```diff
@@ -40,7 +42,7 @@ export default (props: Props) => {
       props.onSend(content);
       return;
     }
-    if (content.trim()) {
+    if (content.trim() || messageEditorService.filesAttachements[props.threadId || 'main']) {
```

Comment:
You must check the object is not empty (length) and the just if the object exists ( ([1, 2] == true) returns false)
Be careful and use a "?.length" (question mark in case it is not defined)

Reply:
Done :+1: